deeprvat/deeprvat/base_configurations.yaml

```diff
@@ -0,0 +1,94 @@
+
+#DeepRVAT model training settings
+training:
+    min_variant_count: 0
+    n_bags: 1
+    drop_n_bags: 0
+    train_proportion: 0.8
+    sample_with_replacement: False
+    n_parallel_jobs: 6
+    dataloader_config:
+        batch_size: 1024
+        num_workers: 0
+        temp_dir: $TMPDIR/deeprvat_train
+        cache_tensors: True
+        chunksize: 100
+
+#Optional Optuna hyperparameter optimizaiton for training
+hyperparameter_optimization:
+    direction: maximize
+    n_trials: 1
+    sampler:
+        type: TPESampler
+        config: {}
+
+#DeepRVAT model settings
+model:
+    type: DeepSet
+    model_collection: agg_models
+    checkpoint: combined_agg.pt
+    config:
+        phi_layers: 2
+        phi_hidden_dim: 20
+        rho_layers: 3
+        rho_hidden_dim: 10
+        activation: LeakyReLU
+        pool: max
+        use_sigmoid: True
+        metrics:
+            objective: MSE
+            objective_mode: min
+            loss: MSE
+            all:
+                MSE: {}
+                PearsonCorrTorch: {}
+                MAE: {}
+                RSquared: {}
+        optimizer:
+            type: AdamW
+            config: {}
+
+#Seed Gene settings
+baseline_results:      
+    correction_method: Bonferroni
```

Comment:
I would move the `correction_method` to the `deeprvat_input_config.yaml` since `baseline_results["alpha_seed_genes"]` and `baseline_results["correction_method"]` are both required for the multiple testing correction settings so splitting them might be confusing.